app/GroupMember.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class GroupMember extends Model
+{
+    public  $timestamps = false;
+
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'groupmembers';
+
+    protected $fillable = [
+        'group_id'
+        ,'username'
+        ,'status_code'
+        ,'action_group_id'
```

Comment:
i don't think action_group_id is necessary for groups @nikoootine unless we plan on saying did the user request to join group or did the group invite the user to join? and even then it should be a status code because we shouldn't mix user id and group id in one column